test_pull_requests

```diff
@@ -837,21 +837,8 @@ popd
         end
       end
       commit = get_last_commit(pull_request['number'], pull_request['base']['repo']['name'])
-
-      changed_after_eval = true
-      if commit['sha'] == previous_sha
-        changed_after_eval = false
-      end
-      #GitHub API isn't consistent here
-      updated_at_str = nil
-      if commit.has_key?('commit')
-        updated_at_str = commit['commit']['committer']['date']
-      else
-        updated_at_str = commit['committer']['date']
-      end
-
-      updated_at = Time.parse(updated_at_str)
-      [updated_at, changed_after_eval]
+      
+      [Time.parse(commit['commit']['committer']['date']), commit['sha'] != previous_sha]
```

Comment:
The comment above about the api being inconsistent makes me suspicious

You also need to add a comment to describe the output now that there are no longer variable names